_includes/featured-dotgov.html

```diff
@@ -6,9 +6,9 @@ <h2 class="usa-alert__heading font-sans-xl">
         Featured .gov
       </h2>
       <h4 class="font-heading-lg text-semibold margin-top-205 margin-bottom-205">
-        <a class="usa-link usa-link--external" rel="noopener noreferrer" href="https://read.gov/">Read.gov</a>
+        <a class="usa-link usa-link--external" rel="noopener noreferrer" href="https://everykidoutdoors.gov/index.htm">EveryKidOutdoors.gov</a>
```

Comment:
https://everykidoutdoors.gov does redirect to the /index.htm page, but I think you should drop `/index.htm` from this href.